starter-code/views/index.ejs

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>My First Map</title>
+  <style>
+    #map {
+      height: 500px;
+      width: 100%;
+    }
+  </style>
+</head>
+
+<body>
+  <h3>My First Map</h3>
+  <div id="map"></div>
+  <div>
+    <ul>
+      <%Place.forEach(place => {%>
+        <li>
+          <%=place.name%>
+        </li>
+    </ul>
+    <% }) %>
+  </div>
+
+  <form class="" action="/" method="post">
+    <input type="text" name="lat" placeholder="Ecris la latitude">
+    <input type="text" name="lng" placeholder="Ecris la longitude">
+    <input type="text" name="name" placeholder="Ecris la longitude">
+    <input type="text" name="description" placeholder="description">
+    <button type="button" name="button">Submit</button>
```

Comment:
I believe you want the button to submit the form. If you do `type="button"` it will not submit. You need to either replace it with `type="submit"` or remove it altogether.